### PR TITLE
New: Add logo to loading page

### PR DESCRIPTION
**Pull request by Snarkenfaugister**
#### Database Migration
NO

#### Description
Add 48px logo to loading screen, above message.
logo

#### Issues Fixed or Closed by this PR

* #4304

### CODE REVIEW

File: frontend/src/Components/Page/LoadingPage.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
   composes: page from '~./Page.css';
 }
+
+.logoFull {
+  margin-top: 50px;
+  margin-right: auto;
+  margin-left: auto;
+  width: 48px;
+  height: 48px;
+}
```
Comment (markus101) [inline frontend/src/Components/Page/LoadingPage.css:11]: Stylelint is failing this due to the lack of newline at the end of the file.

**Review by markus101 — CHANGES_REQUESTED** [review]
Ready to ship once linting passes and CI is green.

**Comment by agneevX** [comment]
This icon is low res on Retina displays.

**Comment by Taloth** [comment]
should've been svg

**Comment by agneevX** [comment]
Almost as if it exists here: https://github.com/Sonarr/Sonarr/blob/phantom-develop/Logo/Sonarr.svg?short_path=b0a7218

**Pull request merged**